code/modules/media/broadcast/transmitters/broadcast.dm

```diff
@@ -50,6 +63,20 @@
 	broadcast() // Bzzt
 
 /obj/machinery/media/transmitter/broadcast/attackby(var/obj/item/W, mob/user)
+	if(panel_open && (istype(W, /obj/item/device/multitool)||istype(W, /obj/item/weapon/wirecutters)))
+		attack_hand(user)
+	if(issolder(W))
```

Comment:
It seemed like an odd choice to me because nanopaste is associated with robots. This seemed more like a mechanical thing to me, so more of a natural association with tools.

And I won't pretend like I'm not pushing an agenda to make solders more useful. They are my pet project, it would be disingenuous to imply that I don't have them on my mind.